test/system/250-systemd.bats

```diff
@@ -294,7 +295,7 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
 }
 
 # https://github.com/containers/podman/issues/13153
-@test "podman rootless-netns slirp4netns process should be in different cgroup" {
+@test "podman rootless-netns pasta processes should be in different cgroup" {
```

Comment:
I looked and looked for some way to do this:
```
    run_podman info --format "what is the current rootless network provider"
    skip if it's anything other than pasta
```
You know, for futureproofing. I could not find any marker. Is there one?

Reply:
Actually the test should happen for slirp and pasta. We do not have a marker it is just some containers.conf setting but looks we never actually explicitly test that here. I don't want to block this PR on this though. I can submit a PR tomorrow to address this and the other comments once this is merged.